skbuild/platform_specifics/windows.py

```diff
@@ -58,31 +58,37 @@ def __init__(self):
                 "https://www.microsoft.com/download/details.aspx?id=8279"
             )
 
-        # For Python 3.5: VS2017, VS2015
+        # For Python 3.5: VS2019, VS2017, VS2015
         elif version.major == 3 and version.minor == 5:
-            supported_vs_years = [("2017", "v140"), ("2015", None)]
+            supported_vs_years = [("2019", "v142"), ("2017", "v141"), ("2015", None)]
```

Comment:
Not sure why this was "v140", but might be needed that way...

Reply:
```suggestion
            supported_vs_years = [("2019", "v142"), ("2017", "v140"), ("2015", None)]
```

Reply:
As far as I figured out, `v140` is needed to build against the toolset of the MSVC version in VS2015.

https://wiki.python.org/moin/WindowsCompilers is not entirely clear about this, but if you look at [Python 3.6.8 for Windows](https://www.python.org/ftp/python/3.6.8/python-3.6.8-embed-amd64.zip), you see it only contains `vcruntime140.dll`. Compare to e.g. [3.9.6](https://www.python.org/ftp/python/3.9.6/python-3.9.6-embed-amd64.zip), which also contains `vcruntime140_1.dll` (which I'm guessing corresponds to `v141`, but as we know, VS/MSVC's version numbering and ABI compatibility are not all too straightforward to understand).

Reply:
So, `("2019", "v142")`: maybe that should also have `v140`? But then it also seems to depend on which version of Python you're building for, if I understood correctly?

Reply:
That's mentioned here: https://cibuildwheel.readthedocs.io/en/stable/faq/#importerror-dll-load-failed-the-specific-module-could-not-be-found-error-on-windows

If you use 2019, there's an extra include, and but you can disable it with a flag.

Reply:
> VS/MSVC's version numbering and ABI compatibility are not all too straightforward to understand 

+1!

Reply:
Ha, you're right. So `vcruntime140_1.dll` does _not_ correspond to `v141`, indeed. Any idea why you had to make the `v140` -> `v141` change, then?

One thing to definitely test is the VS2019 v142 in combination with the oldest Python version available, perhaps? If v142 works on that, then my guess about setting `v140` for backwards compatibility was definitely wrong.

Reply:
Ah, wait. Another online search, and I found this: https://docs.microsoft.com/en-us/cpp/porting/binary-compat-2015-2017?view=msvc-160

> You can mix binaries built by different versions of the v140, v141, and v142 toolsets. However, you must link by using a toolset at least as recent as the most recent binary in your app.